ros2action/test/fixtures/fibonacci_action_server.py

```diff
@@ -56,6 +56,12 @@ def main(args=None):
 
     node = FibonacciActionServer()
     try:
+        # Spend up to 2 seconds doing all the work we need to have the fibonacci
+        # action server actually talking over DDS before signaling to the test it's
+        # OK to start
+        for _ in range(20):
```

Comment:
There may be a way to spin_until_future_complete here, but I just don't know.  Is there a future I can wait on that indicates the action server is up, ready, and can be communicated with by other processes before I print out 'fibonacci_action_server has started' ??